jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/DefaultHandler.java

```diff
@@ -64,6 +64,14 @@ public class DefaultHandler extends Handler.Abstract
 
     public DefaultHandler()
     {
+        super(InvocationType.NON_BLOCKING);
+    }
+
+    public DefaultHandler(boolean serveIcon, boolean showContexts)
+    {
+        this();
+        _serveIcon = serveIcon;
+        _showContexts = showContexts;
```

Comment:
Should be the opposite: the constructor with less parameters calling the one with more parameters, so the defaults are clear.

Reply:
OK.   Should we make them final fields that can only be set with constructor injection?  Guess it doesn't matter as it is not on hot path.